src/Mod/Draft/draftguitools/gui_polararray.py

```diff
@@ -25,37 +25,22 @@
 # \ingroup DRAFT
 # \brief This module provides the Draft PolarArray tool.
 
-from pivy import coin
 from PySide.QtCore import QT_TRANSLATE_NOOP
-
-import FreeCAD as App
 import FreeCADGui as Gui
-import Draft
 import Draft_rc  # include resources, icons, ui files
-import draftutils.todo as todo
-
-from draftutils.messages import _msg, _log
-from draftutils.translate import _tr
 from draftguitools import gui_base
 from drafttaskpanels import task_polararray
 
 # The module is used to prevent complaints from code checkers (flake8)
 bool(Draft_rc.__name__)
 
 
-class PolarArray(gui_base.GuiCommandBase):
+class PolarArray(gui_base.PolarCircularBase):
```

Comment:
You should update the docstring of the Gui Commands mentioning that the parent class, `PolarCircularBase` sets up callbacks that allow selection of edges.